client/src/leap/soledad/client/api.py

```diff
@@ -70,6 +71,26 @@
 """
 SOLEDAD_CERT = None
 
+# A whitelist of modules from where to collect plugins dynamically.
+# For the moment restricted to leap namespace, but the idea is that we can pass
+# other "trusted" modules as options to the initialization of soledad.
+PLUGGABLE_LEAP_MODULES = ('mail', 'keymanager')
+
+
+def collect_plugins(interface):
```

Comment:
Again, the idea of `trusted_modules` seems strange if soledad long-term purpose is to be used outside the context of LEAP/Bitmask. I would instead turn the responsibility of registering the plugins to each module.